_data/projects/Networking_In_Java.yml

```diff
@@ -0,0 +1,18 @@
+---
+name: Networking_In_Java
+desc: A repostiory which goes through implementation of networking protocols in java
+site: https://github.com/PrajjawalBanati/Networking_In_Java
+tags:
+        - java
+        - java-networking
+        - socket-programming
+        - stream-programming
+        - multithreaded-server
+        - tcp-server
+        - udp-server
+        - java-programming
+upforgrabs:
+        name: help wanted
+        link: https://github.com/PrajjawalBanati/Networking_In_Java/labels/help%20wanted
+stats:
```

Comment:
No need to add this part of the file at this time - once this is reviewed and merged it'll get generated based on the activity on GitHub.

Reply:
Ok sir, Thank you so much.